src/components/events/event/volunteer-list/EventVolunteerList.jsx

```diff
@@ -27,12 +27,44 @@ const EventVolunteerList = ({ name, type, eventId, setViewVolunteers }) => {
         volunteerData.push(user);
       }
     });
+
+    const { data: eventWaivers } = await AFCBackend.get(`waivers/${eventId}`);
+    for (let i = 0; i < eventWaivers.length; i += 1) {
```

Comment:
in general, we don't want to use `for` loops when `forEach` for JS arrays exists (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)

`eventWaivers.forEach((waiver, index) => {...}`

You'd still have to modify the array using the index but forEach is generally better syntax in terms of accessing the array elements directly